code/index.html

```diff
@@ -6,28 +6,47 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./style.css" />
     <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Urbanist:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet" />
-    <title>Chat Bot</title>
+    <title>The ice-cream bot &#127846</title>
   </head>
 
+<!--emoji cone ref &#127846 -->
+<!--emoji cup ref &#127848-->
+
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <h1>Welcome to the ice-cream shop &#127846</h1>
+    <h2>Order your favorite ice-cream here!</h2>
+
+    <!--<audio id="audio" autoplay loop controls
+    src="/audio/song.mp3"></audio>-->
+
     <main>
       <section class="chat" id="chat"></section>
-
+      <!-- <section class="bot-msg">
+            <img src="./assets/bot.png" alt="chat bot"/>
+            <div class="bubble bot-bubble">
+              <p>Hello</p>
+            </div>
+            </section>
+          <section class="user-msg">
+            <div class="bubble user-bubble">
+              <p>Hello hello</p>
+            </div>
+            <img src="./assets/user.png" alt="user bot"/>
+          </section> -->
```

Comment:
Hi emma!  if you like you can remove this commented code since is not going to be executed in order to make the file easier to read.